src/11_args.py

```diff
@@ -21,14 +30,21 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(*a))    # Should print 22
 
 # Write a function f3 that accepts either one or two arguments. If one argument,
 # it returns that value plus 1. If two arguments, it returns the sum of the
 # arguments. Google "python default arguments" for a hint.
 
 # YOUR CODE HERE
-
+def f3(*args):
```

Comment:
This is preferably done using default parameter values in the function constructor! Definitely cleaner code that way, do a quick google search on `python default parameters` to see if you can find any ideas to avoid the if/else statements!